SabreTools.DatItems/DatItem.cs

```diff
@@ -610,6 +611,15 @@ public static ConcurrentList<DatItem> Merge(ConcurrentList<DatItem> infiles)
                             saveditem.CopyMachineInformation(file);
                             saveditem.SetName(file.GetName());
                         }
+                        
+                        // add cloneof to parent
+                        // only saveditem add to list (cause dupetype != 0x00)
+                        // used to create parent/child/grand-child chd
+                        // name = parent, cloneof = child, grand-child, more grand-childs (using parent/child)
+                        if (saveditem.Machine.CloneOf == null || !saveditem.Machine.CloneOf.Contains(file.Machine.Name)
```

Comment:
This seems unrelated to the other parts of this PR at least based on the description and title. Please remove these changes or add them to a separate PR.